Source/WebCore/inspector/NetworkResourcesData.cpp

```diff
@@ -80,6 +80,7 @@ unsigned NetworkResourcesData::ResourceData::removeContent()
 unsigned NetworkResourcesData::ResourceData::evictContent()
 {
     m_isContentEvicted = true;
+    setDecoder(nullptr);
```

Comment:
should this also wipe out the certificate if it's been set?

Reply:
The idea to reset the decoder here was because after eviction of content, the decoder is not needed anymore, nothing will be decoded. The certification info can be still used after we reach the resources content memory limit. Do you think we should wipe out the certification info here too? Even it is not counted as resources content?

Reply:
hmm yeah i suppose it doesnt make sense to do that here since we dont track the cost of the certificate info